newIDE/app/src/MainFrame/index.js

```diff
@@ -1893,6 +1891,15 @@ const MainFrame = (props: Props) => {
     }));
   };
 
+  const _onDropEditorTab = (fromIndex: number, toIndex: number) => {
```

Comment:
While (wrongly) used in the file, don't prefix a local variable with an underscore. I think this is because this was adapted in the past from the class component which had underscore to hide private functions - but they are not private by design (because inside the component function, so not exposed by default on its interface, that's one advantage of React hooks/functional components)